.gitignore

```diff
@@ -0,0 +1,11 @@
+# Dependency directories
+/node_modules
+*/node_modules
+
+# lerna files
+/lerna-debug.log
+
+# Lock files
+/package-lock.json
+/yarn.lock
```

Comment:
You should commit these files: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Noted. I've removed it from gitignore.